src/core/geometry/qgsabstractgeometry.h

```diff
@@ -88,7 +88,11 @@ class CORE_EXPORT QgsAbstractGeometry
     //! Segmentation tolerance as maximum angle or maximum difference between approximation and circle
     enum SegmentationToleranceType
     {
+      // Maximum angle between generating radii (lines from arc center
```

Comment:
Can you make these real doxygen tags so that they appear in the generated api docs?